.devcontainer/default/Dockerfile

```diff
@@ -0,0 +1,24 @@
+FROM swiftlang/swift:nightly-main-jammy
+
+RUN \
+  #
+  # Disable apt interactive prompts for this RUN command
+  export DEBIAN_FRONTEND="noninteractive" && \
+    # 
+    # Update apt package list
+    apt-get update && \
+    # 
+    # Install sourcekit-lsp dependencies
+    apt-get install -y \
+      libsqlite3-dev libncurses5-dev python3 ninja-build && \
+    #
+    # Install swift-format
+    apt-get install -y \
+      curl && \
+    mkdir /swift-format && \
+    curl -L https://github.com/apple/swift-format/archive/refs/heads/release/5.9.tar.gz | tar xz --strip-component=1 -C /swift-format && \
+    swift build --package-path /swift-format
+
+ENV \
+  PATH="/swift-format/.build/debug:${PATH}"
```

Comment:
With https://github.com/apple/sourcekit-lsp/pull/1096, the recommended way of running swift-format on the repo is to use the _Run swift-format_ task, so we shouldn’t need this anymore.

Reply:
I think you from the future disagrees :) but yes since `swift-format` is now a dependency this is not strictly necessary
https://github.com/apple/sourcekit-lsp/commit/7c9a6e47e1a999890b88a345f02d31d3e71820a8